src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+export const CHECK_STATUS = 'CHECK_STATUS';
+
+export const initialState = {
+  status: '',
+};
```

Comment:
- [ ] Please set the initial state to be an empty array as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/178458180-0524c76d-df36-4e39-85d4-e9756c04f77c.png)